### PR TITLE
342 built in UI check

### PR DESCRIPTION
**Pull request by chikodich**
Cypress tests to check that all elements are rendered properly depending on the selected UI. Prior login tests also extended for better isolation.

### CODE REVIEW

File: cypress/support/elements/HomepageEls.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+export class HomepageEls {
+  get tokenField() {
+    return cy.get("#token-field");
```
Comment (kilimandzsaro) [inline cypress/support/elements/HomepageEls.js:3]: This and the line 6 is the same. Is there a reason for it?

File: cypress/support/steps/uiCheck.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import LoginPageEls from "../elements/LoginPageEls";
+export class UICheckSteps { 
+  clickStartButton() {
+    LoginPageEls.startBtn.click();
```
Comment (kilimandzsaro) [inline cypress/support/steps/uiCheck.js:4]: missing the `return this`

**Review by kilimandzsaro — CHANGES_REQUESTED** [review]
Since we are not chaining the commands it's ok that the methods in the `uiCheckAssertions.js` file does not have `return this;` here. We usually add the `return this` to every method to be able to chain together the different commands.

File: cypress/support/elements/HomepageEls.js (hunk 1 above)
Comment (chikodich) [reply cypress/support/elements/HomepageEls.js:3]: updated - unnecessary duplicate

File: cypress/support/steps/uiCheck.js (hunk 2 above)
Comment (chikodich) [reply cypress/support/steps/uiCheck.js:4]: updated

Comment (chikodich) [reply cypress/support/steps/uiCheck.js:4]: also updated uiCheckAssertions.js and loginAssertion.js methods for best practice

**Pull request merged**